.mocharc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "timeout": 10000,
+  "recursive": "test/",
+  "exit": true
+}
```

Comment:
When upgrading mocha I had to create the rc file because the good old `.opts` format is no longer supported